tests/test_todo.py

```diff
@@ -208,6 +210,16 @@ def test_is_due(due: datetime.date | datetime.datetime, expected: bool) -> None:
     assert todo.is_due(tzinfo=_TEST_TZ) == expected
 
 
+def test_todo_duration_timespan() -> None:
+    """Test that duration is taken into account in timespan calculation"""
+
+    todo = Todo(
+        dtstart=datetime.datetime(2025, 10, 27, 0, 0, 0, tzinfo=local_timezone()),
```

Comment:
I believe there are existing timespan tests in this file, so can you file the format they are using?

(1) Don't use local_timezone, use a fixed timezone e.g. _TEST_TZ
(2) Don't use timespan to test itself, but instead inspect the `start.isoformat()`, `end.isoformat()` 
(3) Also assert on `duration` since that is relevant to this test

Thank you!